pkg/install/install.go

```diff
@@ -100,12 +101,36 @@ func DefaultImage() string {
 	return image
 }
 
+func validMailAddress(address string) (string, bool) {
+	addr, err := mail.ParseAddress(address)
+	if err != nil {
+		return "", false
+	}
+	return addr.Address, true
+}
+
 func Install(ctx context.Context, image string, opts *Options) error {
 	// I don't want these errors on the screen. Probably a better way to do this.
 	klog.SetOutput(io.Discard)
 	klogv2.SetOutput(io.Discard)
 	utilruntime.ErrorHandlers = nil
 
+	if opts.Config.LetsEncrypt != nil && *opts.Config.LetsEncrypt == "production" {
+		if opts.Config.LetsEncryptEmail == "" {
+			result, err := pterm.DefaultInteractiveTextInput.WithMultiLine(false).Show("Enter your email address for Let's Encrypt (required for production)")
+			if err != nil {
+				return err
+			}
+			opts.Config.LetsEncryptEmail = result
```

Comment:
@iwilltry42 
if email is required when using Let's Encrypt  `production`, should we check email address is valid before setting `opts.Config.LetsEncryptEmail`, and abort installation if not valid ? 

cc: @cjellick

Reply:
We're checking it outside of this `if` statement so it's checking the validity for production **and** staging.
It will error out in both scenarios if the email is invalid: https://github.com/acorn-io/acorn/pull/693/files#diff-3c83c1219d23236aa4c92d672faa69bb463e2001adb87833ca0f7434faded420R127-R131

Reply:
Moved it even further down to always check it if it was set, either via flag (for both prod/staging) or via interactive input (prompt only for prod)